frontend/src/components/Bundle/Bundle.js

```diff
@@ -102,7 +102,12 @@ class Bundle extends React.Component<
             if (!info) return;
             if (info.type === 'file' || info.type === 'link') {
                 return fetchFileSummary(this.props.uuid, '/').then((blob) => {
-                    this.setState({ fileContents: blob, stdout: null, stderr: null });
+                    this.setState({
+                        fileContents:
```

Comment:
I think the cleanest way to do this would be to pass in `responseType: 'text'` when we call `get` from axios in the first place -- then it won't parse as json in the first place.

Reply:
Hi, Ashwin thanks for the suggestion. Just applied.